src/pages/studyView/UserSelections.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { observer } from "mobx-react";
+import { computed } from 'mobx';
+import styles from "./styles.module.scss";
+import { StudyViewFilter } from 'shared/api/generated/CBioPortalAPIInternal';
+import { ChartMeta } from 'pages/studyView/StudyViewPageStore';
+import { isFiltered } from 'pages/studyView/StudyViewUtils';
+
+export interface IUserSelectionsProps {
+    filter: StudyViewFilter;
+    attributesMetaSet: { [id: string]: ChartMeta };
+    updateClinicalDataEqualityFilter: (chartMeta: ChartMeta, value: string[]) => void;
+    clearGeneFilter: () => void;
+    clearCNAGeneFilter: () => void;
+    clearCustomCasesFilter: () => void;
+    clearAllFilters: () => void
+}
+
+
+@observer
+export default class UserSelections extends React.Component<IUserSelectionsProps, {}> {
+
+    constructor(props: IUserSelectionsProps) {
+        super(props);
+    }
+
+    @computed get showFilters() {
+        return isFiltered(this.props.filter)
+    }
+
+    render() {
+        if (this.showFilters) {
+            return (
+                <div className={styles.studyViewUserSelections}>
+                    <span>Your selection:</span>
+                    <div>
+                        {
+                            _.map((this.props.filter.clinicalDataEqualityFilters || []), clinicalDataEqualityFilter => {
+                                let chartMeta = this.props.attributesMetaSet[clinicalDataEqualityFilter.clinicalDataType + '_' + clinicalDataEqualityFilter.attributeId];
+                                return (
+                                    <span className={styles.filter}>
+                                        <span className={styles.name}>{chartMeta.displayName}</span>
+                                        {_.map(clinicalDataEqualityFilter.values, value => {
+                                            return (
+                                                <span className={styles.value}>
+                                                    <span className={styles.label}>{value}</span>
+                                                    <i className="fa fa-times" style={{ cursor: "pointer" }} onClick={event => {
+                                                        this.props.updateClinicalDataEqualityFilter(chartMeta, _.filter(clinicalDataEqualityFilter.values, _value => _value !== value))
```

Comment:
you should make this onClick a member of the component (autobind it).   Try to avoid using anonymous callbacks to handle events in jsx.  It is not performance because diffing algorithm sees it as a new function every render cycle and does unnecessary DOM manipulation.